default/stringtables/en.txt

```diff
@@ -7935,57 +7943,110 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Each launch bay allows a number of fighters to launch each combat round.  Launched fighters take one combat round to reach firing range.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each hit by the strength rating of the shield part. Only the remainder of a weapons damage value will affect the ship. Not all sources of damage will be reduced by a shield, for example [[encyclopedia FIGHTER_TECHS]] are unaffected by ship shields.
+
+A ship shield is not reduced from normal weapons fire, each attack is reduced by the shield value. With a shield strength of 3, 2 shots from [[PC_DIRECT_WEAPON]]s will inflict 1 point of damage each (2 total).'''
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts expand the [[metertype METER_STRUCTURE]] of a ship.
+
 # Ship part classes
 PC_TROOPS
 Ground Troops
 
+PC_TROOPS_DESC
+'''Troop parts integrate with a ship and increase the [[metertype METER_TROOPS]] on-board, which can be used to invade enemy planets. To successfully invade a planet, the total number of invading [[metertype METER_TROOPS]] must exceed those defending (ties are awarded to the defender).
+
+An invading ship is destroyed as part of the landing operation in order to provide safe passage for troops on board. Any additional parts on the ship are lost as well.'''
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on. (An objects [[metertype METER_STEALTH]] must fall within the empires [[metertype METER_DETECTION_STRENGTH]] value to be visible).
```

Comment:
`METER_DETECTION` resolves to `Detection Range`, unless you had some other specification in mind?